os/board/rtl8721csm/src/component/common/api/wifi/rtw_wpa_supplicant/wpa_supplicant/wifi_eap_config.c

```diff
@@ -205,6 +223,12 @@ int eap_start(char *method)
 	}
 #endif
 
+#if CONFIG_ENABLE_FAST
+	if(strcmp(method,"fast") == 0){

```

Comment:
It is better to use `strncmp` instead of `strcmp` for security issue.
How about comparing like below?
```
strncmp(method, "fast", strlen("fast") + 1) == 0
```